arm/Microsoft.ServiceFabric/clusters/.parameters/full.parameters.json

```diff
@@ -0,0 +1,205 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "sxx-az-sfc-full-001"
+        },
+        "tags": {
+            "value": {
+                "resourceType": "Service Fabric",
+                "clusterName": "sxx-az-sfc-x-001"
```

Comment:
This should also be `sxx-az-sfc-full-001`.

Reply:
fixed